src/test/java/com/github/romankh3/image/comparison/ImageComparisonUtilUnitTest.java

```diff
@@ -103,37 +106,38 @@ public void testToBufferedImage() throws IOException {
         assertTrue(bufferedImage instanceof BufferedImage);
         assertTrue(bufferedImage1 instanceof BufferedImage);
     }
+
     @Test
-    public void testGetDifferencePercent()throws IOException{
+    public void testGetDifferencePercent() throws IOException {
```

Comment:
you're using BDD inside the test cases, but not following the same convention for the names. I'd say it should be: `should_calculate_image_difference_percent`. (or without the snake case if you'd like)